templates/ehub.json

```diff
@@ -69,7 +69,7 @@
         "contribRoleAssignmentName": "[guid(uniqueString( variables('resourceGroupId'), deployment().name ))]",
         "subscriptionId": "[subscription().subscriptionId]",
         "tenantId": "[subscription().tenantId]",
-        "webAppStorageAccountName": "[concat(take(parameters('Application Name'), 17), 'storage')]",
+        "webAppStorageAccountName": "[take(concat('alertlogicstorage', uniqueString(parameters('Application Name'))), 24)]",
```

Comment:
Does this need to be globally unique? If so, there's a chance that Application Name will get repeated (between customers, but also between different subscriptions from a single customer)?